Saule/Http/JsonApiAttribute.cs

```diff
@@ -17,13 +17,14 @@ public sealed class JsonApiAttribute : ActionFilterAttribute
         public override void OnActionExecuted(HttpActionExecutedContext context)
         {
             var config = new JsonApiConfiguration();
-            JsonApiProcessor.ProcessRequest(context.Request, context.Response, config, requiresMediaType: false);
 
             if (context.Exception != null)
             {
-                return;
+                throw context.Exception;
```

Comment:
Will the exception still be serialized as a [JSON API error object](http://jsonapi.org/format/#error-objects) when you do this? I don't see how it could be from this code.

Reply:
Great question.  Yes, it is serialized as a JSON API error.  Here's the response when an error occurs inside an API Method, for example:
```c#
        [Route("api/v3/companies")]
        [ReturnsResource(typeof(CompanyResource))]
        public IEnumerable<Company> Get()
        {
            throw new Exception("Watching mark zuckerberg vs congress");
            // ...
        }
```

## Results in the following response
```js
{
  "errors": [
    {
      "title": "Watching mark zuckerberg vs congress.",
      "detail": "   at Saule.Http.JsonApiAttribute.OnActionExecuted(HttpActionExecutedContext context) in C:\\Git\\saule\\Saule\\Http\\JsonApiAttribute.cs:line 23\r\n   at System.Web.Http.Filters.ActionFilterAttribute.OnActionExecutedAsync(HttpActionExecutedContext actionExecutedContext, CancellationToken cancellationToken)\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Filters.ActionFilterAttribute.<CallOnActionExecutedAsync>d__6.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Filters.ActionFilterAttribute.<ExecuteActionFilterAsyncCore>d__5.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__5.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Dispatcher.HttpControllerDispatcher.<SendAsync>d__15.MoveNext()",
      "code": "System.Exception"
    }
  ]
}
```

Reply:
@joukevandermaas see comment above